source/events/pointing.rst

```diff
@@ -0,0 +1,56 @@
+.. include:: ../references.txt
+
+.. _iact-pnt:
+
+.. warning::
+   This is a first draft proposal of a pointing table. Please note that
+   the format is likely subject to change.
+
+
+``POINTING`` extension
+======================
+
+The ``POINTING`` extension is a binary FITS table that contains for a number
+of time stamps the pointing direction of the telescopes. A *pointing* is here
+defined as the centre of the field of view (or centre of the camera
+coordinates). In reality, all telescopes may point to different positions
+(for example for divergent pointing mode).
```

Comment:
It's more than the IRF access, right? It's basically providing the time dependent coordinate transformation for changing between celestial and terrestrial coordinates. Could also be used by tools that do not refer to the IRFs. How about:

> The main purpose of the ``POINTING`` extension is to provide time dependent information on how to transform between celestial and terrestial coordinates.